implemented class Car and class CarWashStation in main.py

motivated2 opened the pull request.

motivated2 committed:
implemented class Car and class CarWashStation in main.py

Y-Havryliv commented on app/main.py:
```diff
@@ -1,8 +1,46 @@
+            if car.clean_mark < self.clean_power:
+                price += (self.calculate_washing_price(car))
+                self.wash_single_car(car)
+        return price
```
it should be rounded to 1 decimal

Y-Havryliv commented on app/main.py:
```diff
@@ -1,8 +1,46 @@
+        )
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
```
You need to check if `clean_power` is greater than `clean_mark` here as well because in case you run this method separately from others it will not work correctly

motivated2 committed:
in class CarWashStation:
-method serve_cars now returns rounded price
-method wash_single_car now checks if clean_power is greater than clean_mark

NEkropo1 reviewed (approved):
Good job!
